### PR TITLE
Expose paused states of streams

### PR DESCRIPTION
**Pull request by dabbler0**
Formerly we detected whether a participant was muted by checking whether the audio track existed. This fails to distinguish between _paused_ audio and _broken_ audio. Since we're modifying connect-call-client now anyway I figured we should expose information about consumer paused states to fix this.

### CODE REVIEW

File: src/RoomClient.ts [hunk 1]
```diff
@@ -341,18 +343,33 @@ class RoomClient {
...
-      this.peers[user.id].consumers[options.label] = consumer;
-      // Screenshare goes in a different stream
-      if (options.label === ProducerLabel.screenshare) {
-        this.peers[user.id].screenshareStream.addTrack(consumer.track);
-      } else {
-        this.peers[user.id].stream.addTrack(consumer.track);
+      const existingConsumer = this.peers[user.id].consumers[label];
+      if (existingConsumer) {
+        if (label === ProducerLabel.screenshare) {
+          this.peers[user.id].screenshareStream.removeTrack(
```
Comment (dabbler0) [inline src/RoomClient.ts:356]: For some reason this unrelated change (remove an existing consumer with the same label) was required to fix a test that failed when I made this change.

**Review by cainlevy — APPROVED** [review]
